### ISSUE TITLE
Capitalisation of progress text

### ISSUE THREAD

**Issue by pawsey-kbuckley** [Pawseyops/maali#122]
Recently posted the output from a Maali build into a jira ticket for a user
and, because of jira's limited display width, saw

```
Using ./qe.cyg for tool configuration information
Warning: non-system build detected, overriding MAALI_TOOL_TYPE (currently apps) and set <Viewport chopped here
downloading the source code for qe 6.7.0
  .. loading PrgEnv-intel/6.0.4
building qe 6.7.0 with intel/17.0.4.196
  .. loading intel/17.0.4.196
...
```

and got confused as to whether the "downloading" was part of the Warming
line above it.

Was thinking that capitalising the "downloading", "building", and the "creating" 
in the line "crreating modules", so to match the lines starting with "Using", 
"Warning" and  "Complete" might be better.

**Comment by marcodelapierre** [comment]
thanks Kevin, will edit these soon!

**Comment by marcodelapierre** [comment]
fixed with commit #baeb550 , thanks Kevin!

**Comment by marcodelapierre** [comment]
will create new release and install it at Pawsey later on
